packages/ckeditor5-table/src/commands/inserttablelayoutcommand.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/**
+ * @module table/commands/inserttablecommand
```

Comment:
This should match the filename.

Reply:
This file will be replaced with the one from editing part.